### PR TITLE
[experiment] disable all svg icons

### PR DESCRIPTION
**Pull request by rasom**
Recording's rate is 30 frames per second 

- - -
On galaxy a500d (slow device), "add new chat" bottom drawer:

* svg: **9** frames after click before animation starts, **5** more frames for finishing animation. Animation duration **0.167s**, delay before starting **0.3s**
* png: **4** frames after click before animation starts, **9** more frames for finishing animation. Animation duration **0.3s**, delay before starting **0.134s**

Animation is almost two times longer with png which means that less frames are dropped, it looks smoother.

- - -
On galaxy a500d (slow device), navigation from chats list to wallet:

* svg: **25** frames (**0.833s**)
* png: **16** frames (**0.533s**)

On galaxy a500d (slow device), navigation from chats list to profile:

* svg: **25** frames (**0.833s**)
* png: **17** frames (**0.567s**)**Navigation takes 30-35% less time when svg icons are disabled.**

- - -
svg icons can be enabled/disabled in profile settings

### CODE REVIEW

**Comment by rasom** [comment]
Similar results on galaxy s9, just a bit harder to notice because app is quite fast there. Opening profile screen takes **2** frames without svg and **5** frames with, **0.067s** vs **0.167s**. But that's barely noticeable by eye. 

"New public chat screen" is **10** frames vs **13** frames (**0.33s** vs **0.43s**) (30fps)

**Comment by annadanchenko** [comment]
used another measurement approach by getting stats with `adb shell dumpsys gfxinfo im.status.ethereum.pr framestats` and cleaning the stats with `adb shell dumpsys gfxinfo im.status.ethereum.pr  reset` see https://developer.android.com/training/testing/performance for details.

Steps:
1. Installed PR build on real Android device Galaxy S6, Android 6.0.1
2. Created account, logged in, opened Chats tab, joined 4 chats, waited for sync to complete
3. run `adb shell dumpsys gfxinfo im.status.ethereum.pr  reset` to clean the stats
4. Tap on + button, wait for menu to be shown (with Start new chat ... etc)
5. Run `adb shell dumpsys gfxinfo im.status.ethereum.pr framestats` to get stats (for png icons, shown as "svg" placeholder instead of every icon)
6. Switched ICONS in profile to get current state with svg icons, switched to Chats and waited for sync to complete
7. run `adb shell dumpsys gfxinfo im.status.ethereum.pr  reset` to clean the stats
8. Tap on + button, wait for menu to be shown (with Start new chat ... etc)
9. Run `adb shell dumpsys gfxinfo im.status.ethereum.pr framestats` to get stats (for current icons)

Got this results:
- png icons (placeholder icon with text "svg" is shown instead of each icon):  https://gist.github.com/annadanchenko/4e55a961197396e38db4dc7f47df8e92
- svg icons (current icons in the app): https://gist.github.com/annadanchenko/bf2db3c16d25c02e558b3e8f77939d0e

In short: 
- png icons (placeholder icon with text "svg" is shown instead of each icon in ui):  
Stats since: 62981072542780ns
Total frames rendered: 19
Janky frames: 18 (94.74%)
90th percentile: 48ms
95th percentile: 53ms
99th percentile: 53ms
Number Missed Vsync: 1
Number High input latency: 0
Number Slow UI thread: 1
Number Slow bitmap uploads: 0
Number Slow issue draw commands: 18


- svg icons (current icons in the app):
Stats since: 63030531496304ns
Total frames rendered: 34
Janky frames: 33 (97.06%)
90th percentile: 44ms
95th percentile: 53ms
99th percentile: 85ms
Number Missed Vsync: 1
Number High input latency: 0
Number Slow UI thread: 6
Number Slow bitmap uploads: 0
Number Slow issue draw commands: 29

**Comment by annadanchenko** [comment]
@andmironov @hesterbruikman @errorists hey design team! Are you ok with replacing current svg icons with png icons (that will give perf improvement on Android)?

**Comment by andmironov** [comment]
i can't help thinking that there is another way to improve the performance. using a non-vercor format for icons feels like a step back, there will be no way to scale those icons, it will require extra work to optimize png icons for @2 and @3 screens, no easy way to change the icon's color. but if it's the only way to optimize the performance, i guess there's no choice :-)

**Comment by errorists** [comment]
hey, yes I'm ok, it's a nuisance sure but a nuisance we can live with. All of the exported icons are optimised and available [over here](https://www.dropbox.com/sh/l32k7cvn6luvs2b/AABf67NOaqSdmuQpmBlg3Ecea?dl=0) 
The rest is up to you guys to do the asset switcheroo :)

**Comment by yenda** [comment]
@rasom did you gave up on that one? seems like it had promising results

**Comment by rasom** [comment]
@yenda I just stuck with modules. This PR is rather POC, so now someone need to take these resources and put them in place. And I still can't do this.

**Comment by StatusWrike** [comment]
➤ Oskar Thoren commented:

Anna DanchenkoIgor MandriginRoman Volosovskyi which project is this in? Sems like it is backlogged for now?

**Comment by oskarth** [comment]
Unito bot doesn't understand that ON HOLD doesn't mean CANCELLED (or maybe it just disagrees)

**Comment by StatusWrike** [comment]
➤ Oskar Thoren commented:

Silly Unito, ON HOLD doesn't mean CLOSE...

**Comment by StatusWrike** [comment]
➤ Oskar Thoren commented:

I guess we need to wrap this GHI to accurately reflect state or something

cc Nabil Naghdy ON HOLD shouldn't close PR https://discuss.status.im/t/replacing-ropsten-with-gorli-network-for-faucet-and-les-needs/1235/12?u=oskarth

**Comment by yenda** [comment]
@oskarth it got closed again

**Comment by mandrigin** [comment]
@yenda it is on purpose, another non-experimental PR will be made later.
